composer/utils/batch_helpers.py

```diff
@@ -0,0 +1,99 @@
+from typing import Any
+
+from composer.core.types import BatchKeys
+
+
+def batch_get(batch: Any, key: BatchKeys) -> Any:
+    """Indexes into the batch given the key.
+
+    Tries all the common combination of batch and key pairs:
```

Comment:
I get where we're going with this list, but I find it a bit confusing on what is getting done here. Perhaps could you add (doctested) examples here?

It might also be worth mentioning that the first operation that succeeds is returned.

Reply:
Done.

Reply:
Removed comment and changed from try except to if else